packages/rest/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "test-project",
+  "version": "1.0.0",
+  "description": "test",
+  "main": "index.js",
+  "author": "jan",
+  "license": "ISC",
+  "scripts": {
+    "start": "node dist/server.js",
+    "dev": "tsnd --respawn src/server.ts",
+    "build": "tsc -p .",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aries-framework/core": "^0.1.0-alpha.266",
+    "@aries-framework/node": "^0.1.0-alpha.266",
+    "body-parser": "^1.19.0",
+    "class-transformer": "^0.3.1",
+    "class-validator": "^0.12.2",
+    "class-validator-jsonschema": "^3.1.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "inversify": "^5.1.1",
+    "multer": "^1.4.3",
+    "ngrok": "^4.1.0",
+    "nodemon": "^2.0.12",
+    "reflect-metadata": "^0.1.13",
+    "routing-controllers": "^0.9.0",
+    "routing-controllers-openapi": "^3.1.0",
+    "side-channel": "^1.0.4",
+    "supertest": "^6.1.6",
+    "swagger": "^0.7.5",
+    "swagger-ui-express": "^4.1.6",
+    "ts-node-dev": "^1.1.8",
+    "tslog": "^3.2.1",
+    "typedi": "^0.10.0",
+    "uuid": "^8.3.2"
```

Comment:
Could you take a good look at all dependencies and see which are required, which are dev dependencies, and which can be updated to newer versions?

e.g. supertest is more a dev dependency. I also see you have two dependency injection libraries installed (inversifyjs and typedi). We use TSyringe for Aries Framework JavaScript....